app/models/lesson.rb

```diff
@@ -0,0 +1,17 @@
+class Lesson < ActiveRecord::Base
+  belongs_to :curriculum
+
+  validates :unit, presence: true,
+    format: { with: /[a-zA-Z]|\d+/, message: "only allows numbers or a single letter" },
```

Comment:
Line is too long. [88/80]<br>Align the elements of a hash literal if they span more than one line.